### PR TITLE
treewide: remove redundant quotes

### PR DESCRIPTION
**Pull request by volth**
remove redundant quotes

### CODE REVIEW

File: nixos/modules/services/mail/postgrey.nix [hunk 1]
```diff
@@ -7,7 +7,7 @@ with lib; let
   natural = with types; addCheck int (x: x >= 0);
   natural' = with types; addCheck int (x: x > 0);
 
-  socket = with types; addCheck (either (submodule unixSocket) (submodule inetSocket)) (x: x ? "path" || x ? "port");
+  socket = with types; addCheck (either (submodule unixSocket) (submodule inetSocket)) (x: x ? path || x ? port);
```
Comment (Mic92) [inline nixos/modules/services/mail/postgrey.nix]: This one does no look right.

Comment (Mic92) [reply nixos/modules/services/mail/postgrey.nix]: It is. My fault.

**Comment by zimbatm** [comment]
@volth what type of tool are you using to do that type of operations?

**Comment by volth** [comment]
> @volth what type of tool are you using to do that type of operations?

https://github.com/lihaoyi/fastparse

https://github.com/NixOS/nixpkgs/pull/63177#issuecomment-502403386

**Comment by zimbatm** [comment]
are you building a library of rewrite rules or are those just one-off operations?

**Comment by volth** [comment]
The rewrite rules are mostly one-liners thank to pattern matching.
I updated the source code of the script on gist.github.com

**Pull request merged**